homeassistant/components/mobile_app/binary_sensor.py

```diff
@@ -0,0 +1,22 @@
+"""Binary sensor platform for mobile_app."""
+from .const import ATTR_SENSOR_TYPE_BINARY_SENSOR as ENTITY_TYPE
+
+from .entity import (async_setup_mobile_app_entry,
+                     async_setup_mobile_app_platform)
+
+DEPENDENCIES = ['mobile_app']
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Set up the mobile app binary sensor."""
+    return await async_setup_mobile_app_platform(ENTITY_TYPE,
```

Comment:
Do we need to support setting up platforms this way or should we just `pass` here?

Reply:
Waiting for clarification from Paulus on what exactly to do, since we are currently calling `async_load_platforms` in webhook.py upon registration.

Reply:
Taken care of, using signals to create new platform entries.